src/build/arg/mod.rs

```diff
@@ -2299,6 +2300,32 @@ impl<'help> Arg<'help> {
         self
     }
 
+
+    /// ... docs ...
+    #[inline]
+    pub fn default_missing_value(self, val: &'help str) -> Self {
+        self.default_missing_values_os(&[OsStr::new(val)])
+    }
+    /// ... docs ...
```

Comment:
Please add an empty line between the functions.

Reply:
Done ... it was pending documentation that I just pushed.